vllm/model_executor/models/qwen.py

```diff
@@ -279,11 +278,18 @@ def load_weights(self,
             for (param_name, weight_name, shard_id) in stacked_params_mapping:
                 if weight_name not in name:
                     continue
-                param = params_dict[name.replace(weight_name, param_name)]
+                name = name.replace(weight_name, param_name)
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
+                    continue
+                param = params_dict[name]
```

Comment:
While it doesn't ned to be addressed right now, can we somehow factor out this part? This seems to be error prone.